docs/concepts/storage/volumes.md

```diff
@@ -990,16 +958,50 @@ several media types.
 ## Out-of-Tree Volume Plugins
 In addition to the previously listed volume types, storage vendors may create
 custom plugins without adding it to the Kubernetes repository. This can be
-achieved by using the `FlexVolume` plugin.
+achieved by using either the `CSI` plugin or the `FlexVolume` plugin.
+
+For storage vendors looking to create an out-of-tree volume plugin, [please refer to this FAQ](https://github.com/kubernetes/community/blob/master/sig-storage/volume-plugin-faq.md) for choosing between the plugin options.
 
+### CSI
+
+CSI stands for [Container Storage Interface](https://github.com/container-storage-interface/spec/blob/master/spec.md),
+a specification attempting to establish an industry standard interface that
+Container Orchestration Systems (COs) can use to expose arbitrary storage systems
```

Comment:
This may be just a nit - but why capitalize "Container Orchestration Systems" and then lower case the 's' in (COs)? I don't see COs used elsewhere, so perhaps remove the capitalization all together and the acronym as well?

Reply:
I think the original intention here was to align with the language in the CSI spec, but since it's not referenced anywhere else, I'll remove the capitalization and acronym.